### ISSUE TITLE
Feature request (maybe): Use Tax info form as a portal for US and CAD users to send their tax docs to Zendesk

### ISSUE THREAD

**Issue by JessBoctor** [Automattic/wp-calypso#79491]
Currently, users who wish to provide documentation for tax-exempt status in CA and US must contact support, who then refer them to sending an email, and then manually add the exempt status

💡  While we can't offer automated tax validation via Taxamo for the US and CA like other countries, maybe we can create some automation in how the contact process goes. 

1. Add form options for US and CAD that ask for any information we need (ie, company name, address etc). Specifically, have an upload field that they can provide their proof (sales certificate, etc)

2. When the form is submitted, there is a notice that we will review their information and reply in 2-3 business days. On the backend, it creates a Zendesk ticket from the information/attachment. The ticket also includes links to the FG page with tax exempt information and how to validate it.

3. HE's manually check the information to make sure it is in line with what we need. If it passes muster, HE's manually set exempt status in Store Admin. If not, they let the customer know what information they are missing. Either way, any follow-up conversation is handled in the ticket created by submitting the form.

The goal of this would be to provide a more uniform experience for our customers and give the HE's a simpler process for directing users who need to submit this information.

**Comment by sirbrillig** [comment]
From our planning call: this sounds like a nice feature request but definitely a larger project. We'll see if we can prioritize it amongst other things.

**Comment by JessBoctor** [comment]
Adding this Slack ping as I think having a clear `billing information` or tax form would have been helpful: https://a8c.slack.com/archives/C03TY6J1A/p1741843285607469

tl;dr Customer moved from the UK to the US and wanted to make sure they wouldn't be charged VAT on renewals. They weren't sure if they completed the `billing information` update since the payment method only takes country/postal code.
